Classes/FairPlayDRM/FPSContentKeyManager.swift

```diff
@@ -0,0 +1,88 @@
+import AVFoundation
```

Comment:
Missing Copyright section above.

Reply:
A job in Jenkins is doing it when merged to develop.